### ISSUE TITLE
Update cheatsheets repositories

### ISSUE THREAD

**Issue by niksy** [chubin/cheat.sh#324]
It seems like latest updates from upstream repositories are available, e.g. if you search for `set` you don’t get all results from https://github.com/tldr-pages/tldr/blob/master/pages/common/set.md. Is this something that needs to be run manually?

**Comment by chubin** [comment]
In my case they are identical. Could you please check if you still have the problem or not?

**Comment by niksy** [comment]
Yes, still getting same problem. Can’t see result for `set -e`, or when searching for `stdout` I don’t get most recent changes from https://github.com/cheat/cheatsheets/blob/master/stdout and they were merged recently.

Is this some caching issue?

**Comment by abitrolly** [comment]
It should be possible to show git revision of the deployed version and also revisions on the cheat sheets at http://cht.sh/

**Comment by chubin** [comment]
My bad, there was a bug indeed. Not it should be fixed, please test.
I've manually triggered the update, so now all the upstream repos should be up-to-date.

The repos updated automatically each hour. So if the fix was successful, upstream changes should be visible after one hour max.

@abitrolly I agree; something like `/:revs` or `/:reps` with all upstream repositories descriptions and their revisions would be ideal.

**Comment by abitrolly** [comment]
Added reminder for `/:sheets` endpoint https://github.com/chubin/cheat.sh/issues/337
